src/pricing-fetcher.ts

```
@@ -64,16 +65,21 @@ export class PricingFetcher implements Disposable {
 
-		try {
-			const fallbackPricing = await this.loadOfflinePricing();
-			logger.info(`Using cached pricing data for ${fallbackPricing.size} models`);
-			return fallbackPricing;
-		}
-		catch (fallbackError) {
-			logger.error('Failed to load cached pricing data as fallback:', fallbackError);
+		const fallbackWrapper = Result.try({
+			try: async () => this.loadOfflinePricing(),
+			catch: error => error,
+		});
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)